web/server/api/endpoints/meta.py

```diff
@@ -22,4 +22,5 @@ def get_api_meta(
     return models.Meta(
         version=_sqlmesh_version(),
         has_running_task=hasattr(request.app.state, "task") and not request.app.state.task.done(),
+        modules=[models.Modules.EDITOR, models.Modules.PLANS],
```

Comment:
What does this do? Is there a way to disable the docs module?

Reply:
and option to read modules from global variable

Reply:
Sounds good. Let's have it be consistent with the other modules so it can be enabled/disabled dynamically.